src/controllers/getsubscribedcontent.js

```diff
@@ -0,0 +1,54 @@
+require('env2')('.env');
+const fetch = require('node-fetch');
+const { CronJob } = require('cron');
+const fs = require('fs');
+
+const query = 'inspiration';
+const newsUrl = `https://newsapi.org/v2/everything?q=${query}&totalResults=10&apiKey=${process.env.NEWS_KEY}`;
+
+var content = {
```

Comment:
do we need this object now? Was this the global one that we decided now not to use any more?